loops_lab.js

```diff
@@ -0,0 +1,325 @@
+
+
+
+console.log("New Version 7/2/2019");
+
+
+// # Loops Lab
+//
+// To complete this lab, do the following:
+//
+// 1. Fork this repo
+// 2. Clone **your** forked repo
+// 3. Create a file called `loops_lab.js` inside your cloned repo
+// 4. Copy the description of the exercise as a comment
+// 5. Underneath the commented description write your code.
+// 6. Ensure your solutions work properly!
+// 7. Open PR against this repo.
+//
+// ___
+//
+// ## Exercises
+//
+// 1. Write a `while` loop and a `for` loop that takes the variable "num" and logs all the numbers, descending, between "num" and 1.
+//
+console.log("#1 -----------------");
+
+let num = 20
+while (num <= 20 && num > 0) {
```

Comment:
You can remove num <=20 in this case from the if statement and it will still work. Something minor though